pkg/manager/manager.go

```diff
@@ -82,58 +82,63 @@ func (rm *Manager) RunUntil(stopCh <-chan struct{}) {
 	go func() {
 		ticker := time.NewTicker(rm.resolution)
 		defer ticker.Stop()
+		rm.Collect(time.Now())
 
 		for {
 			func() {
```

Comment:
no need to have this as a IIFE anymore.  It was only there because of the embedded defer, IIRC.

Reply:
Removed